models/scheduler.go

```diff
@@ -25,8 +25,8 @@ type SchedulerUnit interface {
 	GetName() string
 	VerifyJob(ctx context.Context, namespace NamespaceSpec, job JobSpec) error
 	ListJobs(ctx context.Context, namespace NamespaceSpec, opts SchedulerListOptions) ([]Job, error)
-	DeployJobs(ctx context.Context, namespace NamespaceSpec, jobs []JobSpec, obs progress.Observer) error
-	DeployJobsVerbose(ctx context.Context, namespace NamespaceSpec, jobs []JobSpec) (JobDeploymentDetail, error)
+	// depricated for deploy in optimus
```

Comment:
@Mryashbhardwaj you might want to recheck this comment

Reply:
done